### DISCUSSION TITLE
Hubspot "Custom Objects" are missing associations

### DISCUSSION THREAD

**Discussion by juliaudroc** [airbytehq/airbyte#38650] [Connector Questions]
Hello everyone!

I'm creating a few pipelines of Hubspot data to S3. 

These pipelines include: 
- deals
- contacts
- assets (custom object)

While for deals and contacts I get the respective associations (eg. Deal X is associated with contacts [A,B]), I don't get the same information for the custom object "assets".

Do you know if there is a way around this?
Have you had the same problem?

Thank you!!

**Comment by azorej** [comment]
We use our fork to add associations to custom objects:
https://github.com/airbytehq/airbyte/pull/39376

[a comment by an automated account is not shown]
